ozaria/site/components/teacher-dashboard/BaseMyClasses/index.vue

```diff
@@ -128,8 +128,7 @@
 
 <template>
   <div>
-    <banner-webinar />
-
+    <banner-ho-c />
```

Comment:
```suggestion
    <banner-hoc />
```

Reply:
It is actually intentional. It's auto generated from `BannerHoC` which results in `banner-ho-c`.